pkg/arvo/app/metadata-store.hoon

```diff
@@ -73,15 +89,14 @@
     |=  =path
     ^-  (unit (unit cage))
     ?+  path  (on-peek:def path)
-        [%x %all ~]               ``noun+!>(associated)
-        [%x %group-indices ~]     ``noun+!>(group-indices)
-        [%x %app-name-indices ~]  ``noun+!>(app-name-indices)
-        [%x %app-path-indices ~]  ``noun+!>(app-path-indices)
+        [%y %associations ~]      ``noun+!>(associations)
+        [%y %group-indices ~]     ``noun+!>(group-indices)
+        [%y %app-indices ~]       ``noun+!>(app-indices)
+        [%y %resource-indices ~]  ``noun+!>(resource-indices)
         [%x %metadata @ @ @ ~]
       =/  =group-path  (stab i.t.t.path)
```

Comment:
Noticing this now (late, sry), but just `stab` here is kind of surprising. It works on `(stab '/some/path')`, but that cord isn't actually path-legal, since it contains slashes. (Imagine your frontend sending a path-as-a-string that's `'/outer/some/path/deeper'`. Good luck trying to figure out if the middle thing is supposed to be `/some/path` or `/some` or whatever.)

In my proposal in the email thread, I included `(scot %t path)` for encoding, and `(slav %t part.of.path)` for decoding. Those are crucial, because they turn your `@t` `'/some/path'` into `@ta` `~.~~~2f.some~2f.path`, which _is_ path-legal. You'll run into issues with this if you don't do that safety-conversion.

Reply:
Done, thank you.